src/projetos/CadastroProjetosPage.tsx

```
@@ -0,0 +1,395 @@
+        }
+    }, [])
+
+    const removeImage = () => {
+        setPreviewImage(null)
+        form.setValue("imagem", undefined)
+        if (fileInputRef.current) {
+            fileInputRef.current.value = ""
+        }
+    }
+
+    function onSubmit(values: z.infer<typeof formSchema>) {
```

ESLint

[error] Delete `··`

(prettier/prettier)